packages/web-frontend/src/screens/LandingPage/useAuth.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useSelector } from 'react-redux';
```

Comment:
This util might be unnecessary but i imagine that it might end up having more business logic in it over time